docs/topics/auth/default.txt

```diff
@@ -1006,6 +1019,19 @@ implementation details see :ref:`using-the-views`.
       authenticated users accessing the login page will be redirected as if
       they had just successfully logged in. Defaults to ``False``.
 
+      .. warning::
+
+          If you use this feature attackers might be able to retrieve the
```

Comment:
While we don't normally link to third-party sites, I think it's merited in this case given the complexity of the issue. Suggested text:

If you enable `redirect_authenticated_user`, other websites will be able to determine if their visitors are authenticated on your site by requesting redirect URLs to image files on your website. To avoid this "[social media fingerprinting](https://robinlinus.github.io/socialmedia-leak/)" information leakage, host all images and your favicon on a separate domain.

With the link, I think the code sample could be omitted.